dev/provider_packages/PROVIDER_INDEX_TEMPLATE.rst.jinja2

```diff
@@ -56,20 +56,6 @@ Requirements
 
 The minimum Apache Airflow version supported by this provider package is ``{{ MIN_AIRFLOW_VERSION }}``.
 
-{%- if PREINSTALLED_PROVIDER %}
-
-This provider package is preinstalled by default when Apache Airflow is installed. You do not need to
-install it separately. You can upgrade and downgrade it independently of Apache Airflow package though.
-
-.. note::
-
-    The minimum Apache Airflow version for this package is {{ MIN_AIRFLOW_VERSION }} and it will fail
-    import at runtime if the version of Airflow is lower even if there is no requirement specified in
-    the dependencies - this is because the provider is preinstalled and specifying minimum Apache
-    Airflow version would create a dependency cycle, which confuses dependency tools.
-
-{%- endif %}
-
```

Comment:
Do we need to regenerate the docs to remove this block from the existing docs of those 5 providers? Or it happens by itself as some later step?

Reply:
It is part of the release process: https://github.com/apache/airflow/blob/main/dev/README_RELEASE_PROVIDER_PACKAGES.md#generate-release-notes

There Release Manager will regenerate the docs and increase provider versions.

Reply:
understood was trying to find the usage of the script and guessed that & edited my comment but did not refresh the page to see your comment before. Thank you!